linters/.jshintrc

```diff
@@ -1,26 +1,13 @@
 {
-  "globals": {
-    "__dirname": false,
-    "__filename": false,
-    "console": false,
-    "module": false,
-    "process": false,
-    "require": false
-  },
-  "camelcase": true,
   "bitwise": true,
   "browser": true,
   "curly": true,
   "eqeqeq": true,
   "freeze": true,
-  "immed": true,
-  "indent": 2,
   "latedef": true,
-  "newcap": true,
-  "noempty": true,
-  "quotmark": true,
-  "trailing": true,
+  "node": true,
+  "notypeof": true,
+  "shadow": true,
   "undef": true,
-  "unused": "vars",
-  "maxlen": 80
```

Comment:
FWIW: I think we should have a maxlength defined in JSCS. 100 should probably be good for 2 spaces...or we could keep 128 like in WMA.